modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -235,14 +236,15 @@ public void unassociateObjectField(ObjectField objectField) {
 	@Indexable(type = IndexableType.REINDEX)
 	@Override
 	public ObjectValidationRule updateObjectValidationRule(
-			long objectValidationRuleId, boolean active, String engine,
-			Map<Locale, String> errorLabelMap, Map<Locale, String> nameMap,
-			String outputType, String script,
+			long companyId, long objectValidationRuleId, boolean active,
+			String engine, Map<Locale, String> errorLabelMap,
+			Map<Locale, String> nameMap, String outputType, String script,
 			List<ObjectValidationRuleSetting> objectValidationRuleSettings)
 		throws PortalException {
 
 		_validate(
-			engine, nameMap, outputType, script, objectValidationRuleSettings);
+			companyId, engine, nameMap, outputType, script,
+			objectValidationRuleSettings);
```

Comment:
@mateussgn I believe we can get the companyId from [objectValidationRule](https://github.com/liferay-objects/liferay-portal/blob/ec0bdc7086b4a10dbfd33420c112540a2981678b/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java#L249), this way we can avoid to change a lot of places, and it seems safe to me, what do you think?

Thanks!

Reply:
I agree with you! I'll make the necessary changes.